warehouse/query-core/src/main/java/datawave/query/attributes/DocumentPayload.java

```diff
@@ -0,0 +1,41 @@
+package datawave.query.attributes;
+
+
+import java.util.Map;
+import java.util.TreeMap;
+
+public class DocumentPayload {
+
+    private int count;
+    private boolean trackSizes;
+    private long size;
+    private TreeMap<String, Attribute<?>> dictionary;
+    private long shardTimestamp;
+
+    public DocumentPayload(Document document) {
```

Comment:
As a general rule of thumb, are we serializing the document POJO or its payload?